core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -2387,10 +2387,11 @@ void checkPeriodPredicate(Checker checker) {
             + "WHERE ((`DEPTNO` IN (10, 20)) AND (`GENDER` = 'F'))");
   }
 
-  @Test void testInListEmptyFails() {
-    sql("select * from emp where deptno in (^)^ and gender = 'F'")
-        .fails("(?s).*Encountered \"\\)\" at line 1, column 36\\..*");
-  }
+//  @Test void testInListEmptyFails() {
```

Comment:
I don't think we should remove this test. Most likely the regex is failing because of the lookahead changes.